specification/monitor/resource-manager/readme.md

```diff
@@ -4,34 +4,61 @@
 
 This is the AutoRest configuration file for MonitorClient.
 
-
-
 ---
+
 ## Getting Started
+
 To build the SDK for MonitorClient, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
 
 > `autorest`
 
 To see additional help and options, run:
 
 > `autorest --help`
+
 ---
 
 ## Configuration
 
-
-
 ### Basic Information
+
 These are the global settings for the MonitorClient API.
 
 ``` yaml
 title: MonitorClient
 description: Monitor Management Client
 openapi-type: arm
-
 tag: package-2018-09

```

Comment:
Don't you want to add the new tag here?

Reply:
Initially, I opted not to, since this is a new preview version, but I made the change to generate the preview package by default. This exposed a couple of issues, which I resolved. One of them, however, appears to be a problem with the existing specs, and they will have to eventually be updated (and code changes made) to be compliant with the expected error response format.